python/xgbserver/xgbserver/__main__.py

```diff
@@ -50,4 +50,4 @@
 
         kserve.ModelServer(
             registered_models=XGBoostModelRepository(args.model_dir, args.nthread)
-        ).start([model] if model.ready else [])
+        ).start([model])
```

Comment:
Since model failed to load under /mnt/models, for the fallback case we should pass [] to the start method. The model is loaded from the repository.